ghost/admin/app/components/posts/analytics.hbs

```diff
@@ -44,47 +44,72 @@
     <Tabs::Tabs class="gh-tabs-analytics" as |tabs|>
         {{#if this.post.hasBeenEmailed}}
             <tabs.tab>
-              <h3>Sent</h3>
-              <p>{{format-number this.post.email.emailCount}}</p>
+                <h3>Sent</h3>
+                <p>{{format-number this.post.email.emailCount}}</p>
             </tabs.tab>
 
-            <tabs.tabPanel>Sent</tabs.tabPanel>
+            <tabs.tabPanel>
+                <Posts::PostActivityFeed
+                    @postId={{this.post.id}}
+                    @eventType="sent"
+                    @linkQuery={{hash filterParam=(concat "emails.post_id:[" this.post.id "]") }}
```

Comment:
Note to myself: I'll need to replace the [] in my PR when the filters are updated to always be a single ID.